include/cadmium/celldevs/cell/grid_cell.hpp

```diff
@@ -0,0 +1,50 @@
+/**
+* Copyright (c) 2020, Román Cárdenas Rodríguez
+* ARSLab - Carleton University
+* GreenLSI - Polytechnic University of Madrid
+* All rights reserved.
+*
+* Cells for modeling lattice-based Cell-DEVS scenarios
+*/
+
+#ifndef CADMIUM_CELLDEVS_GRID_CELLS_HPP
+#define CADMIUM_CELLDEVS_GRID_CELLS_HPP
+
+#include <exception>
+#include <cadmium/modeling/ports.hpp>
+#include <cadmium/modeling/message_bag.hpp>
+#include <cadmium/celldevs/cell/cell.hpp>
+#include <cadmium/celldevs/utils/grid_utils.hpp>
+
+
+namespace cadmium::celldevs {
+    /**
+     * @brief DEVS atomic model for defining cells in Cell-DEVS scenarios that are sorted in lattices.
+     * @tparam T the type used for representing time in a simulation.
+     * @tparam S the type used for representing a cell state.
+     * @tparam V the type used for representing a neighboring cell's vicinity. By default, it is set to integer.
+     */
+    template <typename T, typename S, typename V=int>
+    class grid_cell : public cell<T, cell_position, S, V, seq_hash<cell_position>> {
+    public:
+
+        cell_map<S, V> map;     /// Cell map
```

Comment:
/// -> //

Reply:
I don't know if it is standard, but at least in CLion, with ///, comments appear as docstrings. Thus, I use /// for explaining briefly the purpose of the attributes of the main classes

Reply:
<img width="769" alt="Screen Shot 2020-06-02 at 09 20 36" src="https://user-images.githubusercontent.com/12596603/83491664-6b53a800-a4b2-11ea-9123-730b50495b95.png">
Here you can see the different highlighting for // and ///

Reply:
I see, nice.